src/town_cmd.cpp

```diff
@@ -2372,19 +2372,27 @@ static Town *CreateRandomTown(uint attempts, uint32_t townnameparts, TownSize si
 	return nullptr;
 }
 
-static const uint8_t _num_initial_towns[4] = {5, 11, 23, 46};  // very low, low, normal, high
+extern const uint8_t _num_initial_towns[4] = {5, 11, 23, 46};  // very low, low, normal, high
```

Comment:
The `extern` is only used in the other reference to tell the compiler to look elsewhere. 🙂 

```suggestion
const uint8_t _num_initial_towns[4] = {5, 11, 23, 46};  // very low, low, normal, high
```

Reply:
I thought that `extern` was needed on all references to the variable if it's a `const`?